app/containers/markdown/Preview.tsx

```diff
@@ -26,9 +26,9 @@ const MarkdownPreview = ({ msg, numberOfLines = 1, testID, style = [] }: IMarkdo
 	let m = formatText(msg);
 	m = formatHyperlink(m);
 	m = shortnameToUnicode(m);
+	m = removeMarkdown(m);
```

Comment:
Tests?

Reply:
The fact you didn't have to update tests doesn't mean you shouldn't write new regression tests to prevent this from happening in the future.
Also, there's a chance we moved that line around a few years ago, so it's worth checking.

Reply:
I moved those functions together to another file and add unit test for it